blocks/milestone/edit/index.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * WordPress dependencies
+ */
+const ServerSideRender = wp.serverSideRender;
+const { withSelect } = wp.data;
+
+/**
+ * Render Block UI For Editor
+ */
+
+const Milestone = ( { attributes } ) => {
+	return <ServerSideRender block="give/milestone" attributes={ attributes } />;
```

Comment:
Are we using `ServiceSideRender` to also have a shortcode?

Reply:
Yep! A key part of the pitch is making sure that Milestones are available to users as a shortcode as well.

Reply:
Also, server side render protects against broken blocks when the HTML changes.